src/redux/reducers/workspaces-reducer.ts

```diff
@@ -44,6 +44,10 @@ const setWorkspace = (state: WorkspacesStore, { payload }: { payload: { data: Wo
   isLoading: false,
 });
 
+export const isWorkspace = (data: unknown): data is Workspace => {
+  return typeof data === 'object' && data !== null && 'id' in data && 'parent_id' in data && 'type' in data;
```

Comment:
Should we also check what type are the object properties? For example, is this a valid workspaces:
```js
const ws = {
  id: null,
  parent_id: null,
  type: undefined
}

'type' in ws // yields true even if the attribute is undefined or null
```